.gitlab-ci.yml

```diff
@@ -1,2 +1,13 @@
 include:
-  remote: 'https://raw.githubusercontent.com/ebi-ait/gitlab-ci-templates/master/build-release-deploy.yml'
\ No newline at end of file
+  remote: 'https://raw.githubusercontent.com/ebi-ait/gitlab-ci-templates/master/build-release-deploy.yml'
+
+Unit Test:
+  image: quay.io/ebi-ait/ingest-base-images:python_3.6-slim
+  variables:
+    INGEST_API : https://api.ingest.dev.archive.data.humancellatlas.org/
+  script:
+    - apt-get update
+    - apt-get install -y git
+    - pip install -r requirements.txt
```

Comment:
the dev file calls the requirements file.
```suggestion
    
```

Reply:
Removed.